fbw-a32nx/src/wasm/fbw_a320/src/fac/Fac.cpp

```diff
@@ -77,8 +77,22 @@ void Fac::updateSelfTest(double deltaTime) {
   if (selfTestTimer > 0) {
     selfTestTimer -= deltaTime;
     selfTestComplete = false;
+
+    // Hardcoded test light sequence. Between the times (in seconds) in each array, the light is on.
+    selfTestFaultLightVisible = false;
+    // FIXME no references available, best guess
+    double testLightOnTimes[][2] = {{0, 2},       {4.77, 4.87}, {4.9, 5.0},   {5.17, 5.27}, {5.33, 5.43},
```

Comment:
Maybe it makes sense to extract this function somewhere to fbw-common for deduplication now that it's used so many times all over? Also, I think all the `testLightOnTimes` could be `const` or even `constexpr`